package.json

```diff
@@ -138,6 +138,7 @@
     "rollup-plugin-postcss": "3.1.3",
     "rollup-plugin-stylelint": "1.0.0",
     "rollup-plugin-terser": "6.1.0",
+    "rollup-plugin-ts": "^1.4.0",
```

Comment:
We need this plugin so that we produce a single declaration file to match what rollup produces